MetadataExtractor/Extractors/Data/DicomExtract.py

```diff
@@ -0,0 +1,59 @@
+from .IDataExtract import IDataExtract
+from MetadataExtractor.Util import metadataCreation, metadataFormatter
+import pydicom
+from pathlib import Path
+from pydicom._version import __version_info__
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class DicomExtract(IDataExtract):
+    def extract(self, fileInfo):
+        log.info('Extracting metadata for Dicom file "' + fileInfo["file"] + '".')
+        text = ""
+
+        dicom_metadata = pydicom.dcmread(file)
```

Comment:
Change `file` to `fileInfo["file"]`

Reply:
On parsing, I get the following error, please check why this is the case & resolve it: 

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/pydicom/dataset.py", line 2283, in __setitem__
    key = Tag(key)
  File "/usr/local/lib/python3.10/site-packages/pydicom/tag.py", line 122, in Tag
    raise ValueError(
ValueError: Unable to create an element tag from '(0008, 0008)': unknown DICOM element keyword or an invalid int
```